xplique/metrics/fidelity.py

```diff
@@ -178,7 +179,7 @@ def __init__(self,
         self.inputs_flatten = inputs.reshape((len(inputs), self.nb_features, inputs.shape[-1]))
 
         assert 0.0 < max_percentage_perturbed <= 1.0, "`max_percentage_perturbed` must be" \
-                                                      "in ]O, 1]."
+                                                      "in ]0, 1]."
```

Comment:
I propose we stay consistent with the actual code and stay with (the new) f-string way when we can. :)
https://www.python.org/dev/peps/pep-0498/

Reply:
It was just a comment : O instead of Zero